valid-anagram/DaleSeo.rs

```diff
@@ -0,0 +1,21 @@
+use std::collections::HashMap;
+
+// TC: O(n)
+// SC: O(n)
+impl Solution {
+    pub fn is_anagram(s: String, t: String) -> bool {
+        let mut counter: HashMap<char, usize> = HashMap::new();
+
+        for ch in s.chars() {
+            *counter.entry(ch).or_insert(0) += 1;
+        }
+
+        for ch in t.chars() {
+            counter.entry(ch)
+                .and_modify(|cnt| *cnt -= 1)
+                .or_insert(-1);
+        }
+
+        return counter.values().all(|cnt| *cnt == 0)
+    }
+}
```

Comment:
```suggestion
}

```
마지막에 개행문자 깜빡하신 것 같아요! 추가해주세용 😄